rafthttp/streamer.go

```diff
@@ -155,12 +153,11 @@ type WriteFlusher interface {
 type streamWriter struct {
 	to   types.ID
 	term uint64
-	fs   *stats.FollowerStats
```

Comment:
we should not remove this now. we can remove it when we want to deprecate /v2/stats